examples/state_space_connectivity.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 14 13:15:43 2022
+
+@author: jordandrew
+
+For 'mne-connectivity/examples/' to show usage of MEGLDS 
+Use MNE-sample-data for auditory/left
+"""
+
+## import necessary libraries
+import mne
+import numpy as np 
+import matplotlib.pyplot as plt
+from mne_util import ROIToSourceMap, scale_sensor_data #mne_util is from MEGLDS repo
+
+## define paths to sample data
+data_path = '/Users/jordandrew/Documents/MEG/mne_data/MNE-sample-data'
```

Comment:
See how we do it in other examples/tutorials in MNE-Python using `data_path`

Reply:
I have the normal definition for `data_path` (as defined in tutorials) commented out on the next line, I just use this^^ definition so it doesn't keep downloading to my machine.

Reply:
that's not how `data_path` works :)  it's smart enough to look to see if you already have the dataset downloaded; it won't re-download every time.

Reply:
It looks like it wants to download to a different folder than where I keep this project. Can I change the default download path?

Reply:
If you need to work with a local copy because you're modifying the data files multiple times, then you can specify the `path` argument.

See: https://mne.tools/stable/generated/mne.datasets.sample.data_path.html

If you're not touching the data files, I would just leave as is actually cuz you can easily access the path.

Reply:
Yup just found that page :) thanks